test/test-lib/fixtures.ts

```diff
@@ -98,6 +98,37 @@ const loaders: Dictionary<LoaderFunc> = {
 			user,
 		});
 	},
+	releases: async (jsonData, fixtures) => {
+		const user = await fixtures.users[jsonData.user];
+		if (user == null) {
+			logErrorAndThrow(`Could not find user: ${jsonData.user}`);
+		}
+
+		const application = await fixtures.applications[jsonData.application];
+		if (application == null) {
+			logErrorAndThrow(`Could not find application: ${jsonData.application}`);
+		}
+
+		return createResource({
+			resource: 'release',
+			body: {
+				belongs_to__application: application.id,
+				belongs_to__user: user.id,
+				start_timestamp: Date.now(),
+				end_timestamp: Date.now(),
```

Comment:
Did you want these to be identical or is a potentially ms difference fine?

Reply:
I don't see this as a problem, nothing strictly depends on time here.